async-await.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+import {} from 'babel/polyfill';
+import fs from 'fs';
+import path from 'path';
+
+const dir = process.argv[2] || 'input';
+
+// fs module polyfill: this will be unnecessary if fs supports async/await
+function read(filePath) {
+  return new Promise(function(resolve, reject){
+    fs.readFile(filePath, {encoding: 'utf8'}, function(err, result){
+      if (err) { return reject(err); }
+      resolve(result.toString());
+    });
+  });
+}
```

Comment:
Were this a regular npm package, I'd absolutely support the sharing of common code. Since this project is executable documentation, I'd like to avoid internal dependencies. It's nice to be able to view each implementation in its entirety in one file.